src/main/java/org/mockito/internal/stubbing/answers/MethodInfo.java

```diff
@@ -18,7 +17,7 @@
 
     private final Method method;
 
-    public MethodInfo(Invocation theInvocation) {
```

Comment:
Yes at this moment it does, I just changed the arg type by the proper interface. It's the responsability of `MethodInfo` to find the proper element on the invocation, not the caller. Maybe this class should be renamed `InvocationInfo`. I'd rather have some flexibility on this internal class.